README.md

```diff
@@ -219,21 +219,31 @@ If you choose to employ such an alert and have configured kured to
 probe for active alerts before rebooting, be sure to specify
 `--alert-filter-regexp=^RebootRequired$` to avoid deadlock!
 
-### Slack Notifications
-
-If you specify a Slack hook via `--slack-hook-url`, kured will notify
-you immediately prior to rebooting a node:
+### Notifications
 
+When you specify a formatted URL using `--notify-url`, kured will notify
+about draining and rebooting nodes across a list of technologies. 
```

Comment:
What technologies? Is this list documented somewhere?

Reply:
It's in the same Readme:
```
rocketchat:      `rocketchat://[username@]rocketchat-host/token[/channel|@recipient]`

teams:           `teams://token-a/token-b/token-c`

Email:           `smtp://username:password@host:port/?fromAddress=fromAddress&toAddresses=recipient1[,recipient2,...]`

More details here: https://github.com/containrrr/shoutrrr/blob/master/docs/services/overview.md
```